lighthouse-cli/chrome-launcher.js

```diff
@@ -0,0 +1,187 @@
+'use strict';
+
+const childProcess = require('child_process');
+const fs = require('fs');
+const path = require('path');
+const mkdirp = require('mkdirp');
+const inquirer = require('inquirer');
+const net = require('net');
+const rimraf = require('rimraf');
+
+const spawn = childProcess.spawn;
+const execSync = childProcess.execSync;
+
+const platformError = new Error('Platform ' + process.platform + ' is not supported');
+
+function defaults(val, def) {
+  return typeof val === 'undefined' ? def : val;
+}
+
+module.exports = class Launcher {
+  constructor(opts) {
+    opts = opts || {};
+    // choose the first one (default)
+    this.head = defaults(opts.head, true);
```

Comment:
yes. Head means exactly that - "I don't care about where you run, take your most preferred one".

`head` and `select-chrome` have an [inverse relation](https://github.com/GoogleChrome/lighthouse/pull/642/files#diff-87dc5351257b763f95f05a89e01fe573R151) ... "If you want to `select-chrome` don't use your top preference `head`" - something like that.

The appropriate things I could think of were - `head`, `maximum`, `top`, `front`, or `pop` - as it was the top of a priority list.